### PR TITLE
Feature/invoice overviews

### PR DESCRIPTION
**Pull request by nerdsupremacist**

### CODE REVIEW

**Review by thepiwo — APPROVED** [review]
looks good to me

**Review by thepiwo — APPROVED** [review]

File: src/common/resources/translations/translations.de.js [hunk 1]
```diff
@@ -422,11 +422,24 @@ module.exports = {
...
-    'DESCRIPTION': 'Beschreibung'
+    'DESCRIPTION': 'Beschreibung',
+    'NO_LIMIT': 'Kein Limit',
+    'INVOICE_OVERVIEW': 'Übersicht',
+    'SPONSOR_PAYMENTS': 'Meine Zahlungen',
+    'TEAM_PAYMENTS': 'Spenden meiner Sponsoren',
+    'RECIPIENT': 'Empfänger',
+    'PURPOSE': 'Verwendungszweck',
+    'INSTITUTION': 'Institut',
+    'PAID_AMOUNT': 'Bereits Bezahlt',
```
Comment (dominikglandorf) [inline src/common/resources/translations/translations.de.js]: suggest 'Bereits bezahlt'

File: src/common/resources/translations/translations.de.js [hunk 2]
```diff
@@ -422,11 +422,24 @@ module.exports = {
...
+    'TOTAL_AMOUNT': 'Gesamtbetrag',
+    'TRAVELLED_DISTANCE': 'Gereiste Kilometer',
+    'FULFILLED_COUNT': 'Erfüllt',
+    'HAS_PAID': 'Bereits Bezahlt'
```
Comment (dominikglandorf) [inline src/common/resources/translations/translations.de.js]: suggest 'Bereits bezahlt'

File: src/server/views/dynamic/liveblog/highscore.handlebars [hunk 3]
```diff
@@ -77,7 +77,7 @@
                 <tr>
                     <th>{{__ 'PLACE'}}</th>
                     <th>{{__ 'TEAM'}}</th>
-                    <th>{{__ 'DISTANCE'}}</th>
+                    <th>{{__ 'DONATION'}}</th>
```
Comment (dominikglandorf) [inline src/server/views/dynamic/liveblog/highscore.handlebars:80]: Why is this in this PR?

File: src/server/views/dynamic/sponsoring/sponsoring.handlebars [hunk 4]
```diff
@@ -298,73 +302,162 @@
...
+                  <h4>{{__ 'INVOICE_OVERVIEW'}}</h4>
+
+                  <table class="table table-striped text-left" style="min-width: 450px; width: 50%">
+                      <tr>
+                          <td>{{__ 'RECIPIENT'}}</td>
+                          <td>BreakOut e.V.</td>
+                      </tr>
+                      <tr>
+                        <td>IBAN</td>
+                        <td>DE85 7002 2200 0020 2418 37</td>
```
Comment (dominikglandorf) [inline src/server/views/dynamic/sponsoring/sponsoring.handlebars:366]: Should this be public?

**Review by dominikglandorf — CHANGES_REQUESTED** [review]
Looks good, is building, but could not test without populated database.

File: src/server/views/dynamic/sponsoring/sponsoring.handlebars (hunk 4 above)
Comment (nerdsupremacist) [reply src/server/views/dynamic/sponsoring/sponsoring.handlebars:366]: It's sent by email. And is hardcoded in mail service as well. It's not exactly private now

File: src/server/views/dynamic/liveblog/highscore.handlebars (hunk 3 above)
Comment (nerdsupremacist) [reply src/server/views/dynamic/liveblog/highscore.handlebars:80]: Just a detail I noticed. And decided I didn't want to open a PR for a three liner. I can move it to another PR if it bothers you

File: src/common/resources/translations/translations.de.js (hunk 1 above)
Comment (nerdsupremacist) [reply src/common/resources/translations/translations.de.js]: 👍

File: src/common/resources/translations/translations.de.js (hunk 2 above)
Comment (nerdsupremacist) [reply src/common/resources/translations/translations.de.js]: 👍

File: src/common/resources/translations/translations.de.js (hunk 1 above)
Comment (nerdsupremacist) [reply src/common/resources/translations/translations.de.js]: 👍

File: src/common/resources/translations/translations.de.js (hunk 2 above)
Comment (nerdsupremacist) [reply src/common/resources/translations/translations.de.js]: 👍

**Pull request merged**